tests/test_imageCutoutsBackend.py

```diff
@@ -0,0 +1,69 @@
+# This file is part of image_cutout_backend.
+#
+# Developed for the LSST Data Management System.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+import os.path
+import tempfile
+import unittest
+
+import lsst.daf.butler
+import lsst.geom
+import lsst.resources
+import lsst.utils.tests
+from lsst.image_cutout_backend import ImageCutoutBackend, projection_finders, stencils
+
+
+class TestImageCutoutsBackend(lsst.utils.tests.TestCase):
+    @classmethod
+    def setUpClass(cls):
+        try:
+            cls.data_dir = lsst.utils.getPackageDir("testdata_image_cutouts")
+        except LookupError:
+            raise unittest.SkipTest("testdata_image_cutouts not setup.")
```

Comment:
I probably would have done this via a class decorator on the `TestCase`, with import-time logic on `getPackageDir`.  That's not obviously better - this seems fine - but I am curious whether you chose this approach over class decoration for any reason other than habit.

Reply:
Habit.

I remember some refactoring on this point in jointcal tests (where I took this logic from) to get them to be happy with pytest, but the old approach there wasn't quite the same as the approach you describe (it had the decorators on the test methods, not the class).